packages/hebao_v1/contracts/modules/core/ForwarderModule.sol

```diff
@@ -222,6 +215,28 @@ abstract contract ForwarderModule is BaseModule
         );
     }
 
+    function batchCall(
+        address   wallet,
+        address[] calldata to,
+        bytes[]   calldata data
+        )
+        external
+        txAwareHashNotAllowed()
```

Comment:
I think we can skip checking the txAwareHash here, as it is not used in the function body at all. In each `to[i].call`, the calldata is managed using `abi.encodePacked(data[i], wallet, bytes32(0))` and it makes sure each internal transaction will have `bytes32(0)` as the txAwareHash.